telemetry/2.0/README.adoc

```diff
@@ -0,0 +1,34 @@
+---
+title: "MicroProfile Telemetry 2.0"
+date: 2024-06-19
+summary: 
+---
+MicroProfile Telemetry is to adopt OpenTelemetry from CNCF to allow MicroProfile Applications benefit from OpenTelemetry and enable Jakarta RESTful Web Services automatically to be observed if configured.
```

Comment:
My recommended verbiage in italics:

MicroProfile Telemetry adopts OpenTelemetry from the CNCF and adds MicroProfile specification integration for improved MicroProfile application observability.